src/components/NurseryWithdrawals/NurseryWithdrawals.tsx

```diff
@@ -251,9 +242,13 @@ export default function NurseryWithdrawals(): JSX.Element {
                 value={searchValue}
                 onChange={(value) => setSearchValue(value as string)}
               />
-              <IconButton onClick={handleFilterClick} size='medium' className={classes.iconContainer}>
-                <Icon name='filter' className={classes.icon} size='medium' />
-              </IconButton>
+              <Button
+                id='filterNurseryWithdrawal'
+                onClick={() => handleFilterClick}
```

Comment:
I think the onClicks are broken.. you can change this to
`onClick={handleFilterclick}` or `onClick={() => handleFilterClick()}`

Reply:
Oops, missed this thanks!